pkg/event/target/mqtt.go

```diff
@@ -57,6 +59,9 @@ func (m MQTTArgs) Validate() error {
 	default:
 		return errors.New("unknown protocol in broker address")
 	}
+	if m.QueueDir != "" && !filepath.IsAbs(m.QueueDir) {
```

Comment:
what happens here if `validate` returns error?

Reply:
@kannappanr We cannot set an invalid config. The err from Validate() dis-allows setting the new config. The `config.Validate()` covers all the `target.Validate()` for the targets.

Reply:
@kannappanr The `mc set config` will validate the config. Which will reflect for errors.

![image](https://user-images.githubusercontent.com/5410427/51082462-58a9c900-172d-11e9-86c1-997c6f2e3134.png)

Reply:
ok.